doc/src/challenges/0007-atomic-types.md

```diff
@@ -0,0 +1,83 @@
+# Challenge 7: Safety of Methods for Atomic Types and `ReentrantLock`
+
+- **Status:** Open
+- **Tracking Issue:** [#83](https://github.com/model-checking/verify-rust-std/issues/83)
+- **Start date:** *2024-09-09*
+- **End date:** *2024-12-10*
+
+-------------------
+
+## Goal
+
+Verify the safety of:
```

Comment:
We might lack consistency here, but I am not sure that "safety" is unambiguously defined, let alone that all potential readers know what we mean by it? With "absence of undefined behavior" that would be something that one can search for with the help of their favorite search engine, but "safety" alone won't turn up useful hits.

Reply:
I use "safety" interchangeably with "no undefined behavior"--"safety" is just a more concise way to say it.
However, I realized after reading your comment that this definition comes from the [interactive book](https://rust-book.cs.brown.edu/ch04-01-what-is-ownership.html?) I used to learn Rust, and that the official book does not state that equivalence. So, it's probably unreasonable to expect that everyone else has the same definition as I do. I'll add a sentence at the top stating this assumption so that I can say "safety" throughout and keep the meaning clear.